lightning/backend/local.go

```diff
@@ -77,6 +79,7 @@ const (
 
 	defaultPropSizeIndexDistance = 4 * 1024 * 1024 // 4MB
 	defaultPropKeysIndexDistance = 40 * 1024
+	IndexEngineID                = -1
```

Comment:
this const is not used in package `backend`, so there is no need to move it here

Reply:
Yes....I changed it for other reason and it does need now. I will revert it.